### PR TITLE
feat: Allow specifying height for ui.inline #1780

### PR DESCRIPTION
**Pull request by marek-mihok**
This change adds the `height` prop for ui.inline component.
```ts
interface Inline {
  /** The components laid out inline. */
  items: Component[]
  /** Specifies how to lay out the individual components. Defaults to 'start'. */
  justify?: 'start' | 'end' | 'center' | 'between' | 'around'
  /** Specifies how the individual components are aligned on the vertical axis. Defaults to 'center'. */
  align?: 'start' | 'end' | 'center' | 'baseline'
  /** Whether to display the components inset from the parent form, with a contrasting background. */
  inset?: B
  /** Custom height  in `px`, `vh` or `rem`. Use '1' to fill the height of the remaining space. E.g. '100vh', '300px' or '1'. */
  height?: S
}
```
As API description says, the accepted values are the following CSS units: `px`, `vh` or `rem`. It is also possible to fill the all remaining height of the parent component by specifying `1`.

Resolves _ui.inline height_ part of the #1780.

Also regression tests are adjusted since this change defines `display: flex` for cards and could lead to empty space in screenshots. There are 2 changes for regression tests:

- the height is defined for the generated images
- the screenshot of the widget element is taken instead of the form element if there is only one widget in the form

### CODE REVIEW

File: ui/src/form.tsx [hunk 1]
```diff
@@ -175,6 +175,8 @@ interface Inline {
   align?: 'start' | 'end' | 'center' | 'baseline'
   /** Whether to display the components inset from the parent form, with a contrasting background. */
   inset?: B
+  /** Custom height  in `px`, `vh` or `rem`. Use '1' to fill the height of the remaining space. E.g. '100vh', '300px' or '1'. */
```
Comment (mturoci) [inline ui/src/form.tsx:178]:
```suggestion
  /** Height of the inline container. Accepts any valid CSS unit e.g. '100vh', '300px'. Use '1' to fill the remaining card space. */
```

File: ui/src/form.tsx [hunk 2]
```diff
@@ -252,6 +266,8 @@ export const
       justify={m.justify || 'start'}
       align={m.align || 'center'}
       inset={m.inset}
+      // Checks for '1' or positive float value ending with 'px', 'rem' or 'vh'.
+      height={/^\d+(\.\d+)?(px|rem|vh)$|^1$/.test(m.height || '') ? m.height : undefined}
```
Comment (mturoci) [inline ui/src/form.tsx:270]: This is unnecessary, the docs should be enough.
```suggestion
      height={m.height}
```

**Review by mturoci — CHANGES_REQUESTED** [review]
Thanks @marek-mihok. Can you also add some useful examples showcasing when can this be helpful to the docs?

**Comment by marek-mihok** [comment]
@mturoci sure. Also I will add a better example (login page) once the direction is implemented for ui.inline.

**Comment by mturoci** [comment]
> Also I will add a better example (login page) once the direction is implemented for ui.inline.

It would be better to have examples for both, height-only and height + direction.

**Review by mturoci — APPROVED** [review]
LGTM. @lo5 can you do a quick API review, please? Thanks!

**Review by lo5 — APPROVED** [review]
